src/nvim/ops.c

```diff
@@ -126,6 +127,30 @@ static char opchars[][3] =
   { Ctrl_X, NUL, false },    // OP_NR_SUB
 };
 
+char *nvim_lltoa(int64_t val, int base)
```

Comment:
Delete. only used for counting length, we can have a static function just for that for now.